addons/crm/tests/test_crm_lead.py

```diff
@@ -758,3 +758,69 @@ def test_phone_mobile_update(self):
         self.assertEqual(lead.phone, self.test_phone_data[1])
         self.assertEqual(lead.mobile, self.test_phone_data[2])
         self.assertFalse(lead.phone_sanitized)
+
+    @users('user_sales_manager')
+    def test_crm_lead_alias_help(self):
```

Comment:
Can you move this after `test_crm_team_alias` and name it `test_crm_team_alias_helper` or something like that.

Trying to keep related tests close together, and they need to be in alphabetical order so they're easier to locate and are executed in the order they appear :)